### PR TITLE
Create Blogging On Nostr

### PR DESCRIPTION
**Pull request by BrutusBondBTC**
Added A blogpost.  I hosted the images on nostr.build. I'm not sure if I should upload them to github instead.

### CODE REVIEW

**Comment by bitkarrot** [comment]
Blog looks awesome! @BrutusBondBTC 

BTW, you forgot the metadata and the post name should replace spaces with _ and end in ".md" 

Example Metadata from https://raw.githubusercontent.com/plebnet-dev/website/main/src/content/post/crafting_your_story.md
---
publishDate: 2023-10-07T20:22:00-05:00
title: Crafting Your Story - Paving Your Way to a Bitcoin Career
author: Brandon Sauceda
description: Learn how to share your unique journey into Bitcoin.
image: /images/bitcoin_community_growth.png
category: Education
tags:
  - Bitcoin
---

If you need some tag suggestions, @saucy-tech 

Since @bliotti is working on images on the other PR, I can merge this one in without and we'll add the images later.

File: src/content/post/Blogging On Nostr [hunk 1]
```diff
@@ -0,0 +1,113 @@
+
```
Comment (bitkarrot) [inline src/content/post/Blogging On Nostr:1]: Add metadata to the top of the blog post here.

**Review by bitkarrot — CHANGES_REQUESTED** [review]

**Review by saucy-tech — CHANGES_REQUESTED** [review]
This is great. Yes please include the frontmatter. I'd recommend the Opinion category and a nostr tag.

```
---
publishDate: 2023-10-07T20:22:00-05:00
title: Your Blog Post Title
author: Your Name
description: Short description on blog
image: /images/your_image_here.png
categories:
  - Opinion
tags:
  - nostr
---
```

**Comment by BrutusBondBTC** [comment]
Thanks. I didn't even know about tags, metadata or frontmatter.

**Comment by bitkarrot** [comment]
> Thanks. I didn't even know about tags, metadata or frontmatter.

No worries, This branch will need to be updated, please see comment from https://github.com/plebnet-dev/website/pull/72#issuecomment-1788282520

**Review by BrutusBondBTC — COMMENTED** [review]
I redid the post, added the metadata and changed a few typos, but now I'm not sure how to submit it again.

**Comment by bitkarrot** [comment]
@BrutusBondBTC how about, close this PR and make a new one? might be easier for you.

**Comment by bitkarrot** [comment]
bump. @BrutusBondBTC

**Comment by bitkarrot** [comment]
OK I see the metadata, please also edit the file name on your branch so that it has no spaces and ends with .md , Example "Blogging_on_Nostr.md"

@amsalmeron invited you as reviewer, please help confirm that the blog post displays correctly.


Screenshot 2023-11-07 at 11 44 02 AM

**Comment by bitkarrot** [comment]
@BrutusBondBTC you can 1-click edit on github web

Screenshot 2023-11-07 at 11 50 11 AM

**Comment by BrutusBondBTC** [comment]
Ok. I’ll try this when I get home from the grocery store.

**Comment by BrutusBondBTC** [comment]
I think I did this. Did it work?

**Comment by bitkarrot** [comment]
> I think I did this. Did it work?

It still has the old name without the "_'' and the .md ending on this branch.

**Comment by bitkarrot** [comment]
I updated it for you, and will merge in a bit

**Review by bitkarrot — APPROVED** [review]

**Pull request merged**